web/src/main/bvmui/src/styles/_variables.scss

```diff
@@ -1,5 +1,5 @@
 $colors: (
-  'primary': #4a4e4d, 
+  'primary': #61b6a1, 
```

Comment:
This color seems to not fit with the overall UI style. 
Can we choose something darker to better contrast with the white characters? @mino181295  do you have any suggestion?